Networking/Networking/Extensions/CodingUserInfoKey+Woo.swift

```diff
@@ -24,4 +24,8 @@ extension CodingUserInfoKey {
     /// Used to store the UserID within a Coder/Decoder's userInfo dictionary.
     ///
     public static let userID = CodingUserInfoKey(rawValue: "userID")!
+
+    /// Used to store the Granularity within a Coder/Decoder's userInfo dictionary.
+    ///
+    public static let granularity = CodingUserInfoKey(rawValue: "granularity")!
```

Comment:
just wondering, are all these static keys used in the same network call?

Reply:
Not really. These are used to inject certain values into the payload returned by a network response. For example, imagine the api that returns the list of orders does not include the siteID, but we want to have that value in the model objects that result after parsing the response. 

We use these keys to inject variables like that into the response (you can see this particular one being used in OrderStatsV4Mapper:

```
    func map(response: Data) throws -> OrderStatsV4 {
        let decoder = JSONDecoder()
        decoder.userInfo = [
            .siteID: siteID,
            .granularity: granularity
        ]
        return try decoder.decode(OrderStatsV4Envelope.self, from: response).orderStats
    }
```